docs/content_management/field_types/field_type_storage.md

```diff
@@ -178,3 +178,52 @@ Also note that there can be several gateways per Field Type (one per storage eng
 !!! tip
 
     Gateway configuration for built-in Field Types is located in [`core/src/lib/Resources/settings/storage_engines/`](https://github.com/ibexa/core/tree/main/src/lib/Resources/settings/storage_engines).
+
+## Storing Field Type settings externally
+
+Just like in the case of data, storing [Field Type settings](type_and_value.md#field-type-settings) 
+in Content Item tables may prove insufficient. 
+It is not a problem if your setting specifies the allowed number of characters 
+in a text field. 
+However, the Field Type may represent a more complex object, for example, it may 
+consist of two or more other fields, such as the name, SKU and price, and there 
+can be a set of default values instead of just one. 
+Once you add validation rules for these field values, then it becomes an issue.
+    
+You can overcome this obstacle: 
+When you create a new Field Type, you can move Field Type settings 
+to external storage.
+    
+!!! note
+    
+    Another benefit of an external storage is that there can be database relations to 
+    other objects/entities, and the database itself can maintain the integrity of data.
+    
+First, implement the `FieldConstraintsStorage` interface:
+    
+```php
+interface FieldConstraintsStorage
+    {
+        public function storeFieldConstraintsData(
+            int $fieldDefinitionId,
+            FieldTypeConstraints $fieldTypeConstraints
+        );
+    
+        public function getFieldConstraintsData(
+            int $fieldDefinitionId
+        ): FieldTypeConstraints;
+    
+        public function deleteFieldConstraintsData(
+            int $fieldDefinitionId
+        ): void;
+    }
+```
```

Comment:
This interface already exists, so what is actually expected from the developer is to create a class implementing said interface.

You should use it's fully qualified class name to avoid confusion / having to look up for this interface in code.